capnp/Cargo.toml

```diff
@@ -40,3 +40,10 @@ std = ["embedded-io?/std"]
 # message readers to be `Sync`. Note that AtomicUsize is not supported by all
 # rustc targets.
 sync_reader = []
+
+#[lints]
+#workspace = true
```

Comment:
Why is this commented out? I guess the idea is that we intend to delete the `lints.clippy` block below, and when that happens, we'll uncomment this?